### ISSUE TITLE
A few user interface issues on Mac

### ISSUE THREAD

**Issue by Eric-Wainwright** [CamWagner/lizzie#229]
1)  When opening or saving files, you can’t drill into subfolders.  Also, double-clicking doesn’t open a file.

2) Pressing the Home button goes to the beginning of a game, but doesn’t refresh the board.  You need to hit the down arrow to refresh the board.

3) Sometimes, opening a file over another game has the same refresh issue as #2.  You need to hit the down arrow to refresh the board.

4) The mouse wheel is overly sensitive while playing games.  Clicking on an intersection will sometimes scroll back many moves.  (I had a similar issue with Sabaki which was eventually resolved.  You can read about it here: https://github.com/SabakiHQ/Sabaki/issues/200).

**Comment by featurecat** [comment]
what do you mean by drilling into subfolders?

**Comment by raspingbard** [comment]
What I mean is that the File Open dialog is stuck on my root user folder.  I can't open any subfolders because the Open button is grayed out.  Thus, I have to put all my SGF files in the root user folder.

<img width="551" alt="file open dialog" src="https://user-images.githubusercontent.com/28121941/40322312-348d6d22-5cef-11e8-92c8-4cf876480196.png">

**Comment by dfannius** [comment]
For what it's worth, I use Leela on macOS (both Sierra and High Sierra) and have not experienced issue 1.

**Comment by featurecat** [comment]
I notice some folders are grayed out and others look clickable. Why is that? Do they work? Is that because they contain sgfs?

**Comment by raspingbard** [comment]
@dfannius - I use Leela on MacOS too and don't have any problems.  It's only with Lizzie and Leela Zero.  Of course, Leela is based on Qt (a cross-platform framework) and Lizzie is based on Java.

@featurecat - the grayed out folders are links (little arrow in lower left of icon).  Those don't open either.

**Comment by dfannius** [comment]
Whoops, I apologize, I meant Lizzie.

**Comment by dfannius** [comment]
Whoops, I apologize, I meant Lizzie.

**Comment by raspingbard** [comment]
@dfannius - Hmmm, strange that you don't have a problem opening folders.  I'm running High Sierra v10.13.2 and Java v10.0.1.  What version of Java are you running?

**Comment by dfannius** [comment]
High Sierra 10.3.4.

```
java version "1.8.0_151"
Java(TM) SE Runtime Environment (build 1.8.0_151-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.151-b12, mixed mode)
```

**Comment by raspingbard** [comment]
Dan - did you mean High Sierra 10.13.4 or something else?  That version number doesn't seem right.

I believe you're running Java SE 8 where I'm running Java SE 10 (the latest).  I wonder if that could be the difference?

**Comment by dfannius** [comment]
Yes, High Sierra 10.13.4. Comment corrected again. Apologies again for being so sloppy in this thread.

I wouldn't be surprised if the differing Java versions are responsible but for obvious reasons I'm reluctant to upgrade and find out!

**Comment by bittsitt** [comment]
I'm on java 10 (and mac 10.13.4), do not see the problem here.

java 10 2018-03-20
Java(TM) SE Runtime Environment 18.3 (build 10+46)
Java HotSpot(TM) 64-Bit Server VM 18.3 (build 10+46, mixed mode)

**Comment by raspingbard** [comment]
I had some progress.  I upgraded to the latest High Sierra 10.13.4 and the refresh problems #2 and #3 went away.  So that's good.

The file open problem still remains (#1).  I tried editing the code to play around with chooser.showOpenDialog method in LizzieFrame.java.  I stripped out the filter and last-folder options on the chooser object, but nothing seemed to fix the problem.
